fix for hanging mdgx test (Test.1p7e) on long installation paths, backported from AmberTools 16
author: David Cerutti <dscerutti@gmail.com>
--- amber14/AmberTools/src/mdgx/Manual.c.orig	2016-06-10 11:39:47.719576723 +0200
+++ amber14/AmberTools/src/mdgx/Manual.c	2016-06-10 11:41:16.670677715 +0200
@@ -142,9 +142,10 @@
   endfound = 0;
   j = 0;
   while (endfound == 0) {
+    vpivot = j + width;
     for (i = 0; i < width; i++) {
       if (vpar[j+i] == ' ') {
-	i++;
+        i++;
         vpivot = j+i;
       }
       if (vpar[j+i] == '\0') {
